winrm/x509_test.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2016 Canonical ltd.
+// Copyright 2016 Cloudbase solutions
+// licensed under the lgplv3, see licence file for details.
+
+package winrm_test
+
+import (
+	"io/ioutil"
+	"os"
+
+	"github.com/juju/utils/winrm"
+
+	gc "gopkg.in/check.v1"
+)
+
+var (
+	base = "/tmp/base"
+	cert = "winrmcert.pem"
+	key  = "winrmkey.pem"
+)
+
+func (w *WinRMSuite) TestLoadClientCert(c *gc.C) {
+	cert := winrm.NewX509()
```

Comment:
`gocheck.C` has a method called MkDir that will create a temporary dir, and return its path. Please use that instead of the hard-coded "/tmp/base".